src/Game.py

```diff
@@ -14,16 +14,26 @@ def __init__(self):
 
     def runGame(self):
         pygame.init()
+        clock = pygame.time.Clock()
         pygame.display.set_caption('Basic Pygame program')
         self.isRunning = True
         while self.isRunning:
+            dt = clock.tick(clock.get_fps())/1000
             for event in pygame.event.get():
-                if event.type == QUIT or event.type == KEYDOWN:
+                if event.type == QUIT:
                     self.isRunning = False
+                elif event.type == KEYDOWN:
+                    if event.key == K_ESCAPE:
+                        self.isRunning = False
+            man.propagate(self.screen,(120*dt))           
             self.screen.blit(self.background, (0, 0))
+            pygame.draw.aaline(self.screen, (0,0,0), (0,line_height), (600,line_height))
+            man.drawCharacter(self.screen)
             pygame.display.flip()
         pygame.quit()
 
-
+line_height = 450
```

Comment:
Even if we use line_height, we must declare inside __init__ making it  a class variable